code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -587,7 +587,7 @@
 	results = list(/datum/reagent/exotic_stabilizer = 2)
 	required_reagents = list(/datum/reagent/plasma_oxide = 1,/datum/reagent/stabilizing_agent = 1)
 
-/datum/chemical_reaction/silverolidification
+/datum/chemical_reaction/silversolidification
```

Comment:
```suggestion
/datum/chemical_reaction/silver_solidification
```
Damn, they had one job. One job!

Reply:
![image](https://user-images.githubusercontent.com/24975989/96194095-8db9ff80-0f41-11eb-8c8e-97a4ba03ff0b.png)

Do you still want me to change this? As it is inconsistent with our other solidification reaction type paths.

Reply:
Better to have one inconsistent addition that follows the contribution guidelines (which state that type paths must be snake case), thanperpetuateabadpatternofsavingonthenumberofspaces,makingthewholethingmuchlessreadable.

Reply:
Well, since they're all here in this one file, I decided to expand (OH NO MY ATOMISATION REE) the scope of this PR to snekify them all.